src/block/examples.js

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+
+import React from 'react';
+import {Block} from './';
+import examples from './examples-list';
+
+export default {
+  [examples.DEFAULT]: function Story1() {
+    const colors = ['primary', 'negative', 'warning', 'positive'];
+    const elements = [];
+
+    for (const color of colors) {
+      for (let x = 100; x <= 700; x += 100) {
+        const colorString = `${color}${x}`;
+        elements.push(<Block color={colorString}>{colorString}</Block>);
+      }
+    }
+
+    return elements;
+  },
+  [examples.FONTS]: function Story2() {
+    const sizes = [
+      100,
+      200,
+      250,
+      300,
+      350,
+      400,
+      450,
+      500,
+      600,
+      700,
+      800,
+      900,
+      1000,
+    ];
+    const elements = [];
+
+    for (const size of sizes) {
```

Comment:
can we simply do a `const elements = sizes.map(...)`?

Reply:
Good catch. I was able to just directly return the `.map()`